### ISSUE TITLE
[Feature Request] Live preview of Shutter Speed and ISO values

### ISSUE THREAD

**Issue by mohdmujahidansari526** [troop/FreeDcam#121]
This app is almost perfect but just lacks the live preview of Shutter Speeds, ISOs values so its kinda difficult to measure out that what kind of output the current Manual settings are gonna do.
Do have a look and enable this if possible.

**Comment by troop** [comment]
thats up to the manufacture on camera1. there is no way to direct control it.
camera2 is different there the preview shows what you get

**Comment by mohdmujahidansari526** [comment]
if i select camera2api in settings then it'll show live previews of manual camera settings??

**Comment by troop** [comment]
yep up to 0,8 sec. after that i blocked it cause it gets to laggy.

**Comment by mohdmujahidansari526** [comment]
can u enable more open shutter timing for Redmi 3S?

**Comment by troop** [comment]
no its limited due hal

**Comment by mohdmujahidansari526** [comment]
alright.
